File: csharp/Svix/Management.cs
// this file is @generated
#nullable enable
using Microsoft.Extensions.Logging;
using Svix.Models;

namespace Svix
{
    public class Management(SvixClient client)
    {
        readonly SvixClient _client = client;

        public ManagementAuthentication Authentication
        {
            get => new ManagementAuthentication(_client);
        }
    }
}

File: java/lib/src/main/java/com/svix/api/Management.java
// this file is @generated
package com.svix.api;

import com.svix.SvixHttpClient;

import lombok.Getter;

public class Management {
    private final SvixHttpClient client;

    @Getter private final ManagementAuthentication authentication;

    public Management(SvixHttpClient client) {
        this.client = client;
        this.authentication = new ManagementAuthentication(client);
    }
}
